.env.dist

```diff
@@ -47,14 +40,13 @@ INVITATION_SALT=salt
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
 DELIVERY_ADDRESS=your_dev@email.com
-DELIVERY_NAME=Caesar
+COMPANY_NAME='Local Caesar'
```

Comment:
Use “Caesar App” instead of.

Reply:
Also I suggest use “APP_NAME” instead of “COMPANY_NAME”

Reply:
done

Reply:
> APP_NAME

done